Library/Homebrew/extend/ENV/shared.rb

```diff
@@ -241,6 +241,8 @@ def determine_cc
   # Currently only used by aalib in core.
   sig { void }
   def ncurses_define
+    odeprecated "ENV.ncurses_define"
```

Comment:
```suggestion
    # odeprecated "ENV.ncurses_define"
```

and then I can remove it in https://github.com/Homebrew/brew/pull/10397 when we're ready for a minor release?

Reply:
Ok, done.